### ISSUE TITLE
not really an issue...just a suggestion

### ISSUE THREAD

**Issue by Techtronik** [pahimar/Equivalent-Exchange-3#9]
sorry if all of this is a little too long i have trouble keeping my writing compact

TLDR:
damage value for dm/rm tools, armor, and weapons from the current recipes with appropriate damage values for the abilities. no damage value tools craftable from the damage value items and needed for the next tier of items (dm damageable needed for dm inf needed for rm damageable needed for rm inf). i could also suggest a damage value for the handheld mass destruction items like the destruction catayst. these would lower the 'op-ness' and the low damage value on the high destruction items would make them less appealing.

Does this sound stupid? does it sound good? does it sound tedious? i think it may be too far as i've explained it but the idea could be throttled back a bit using less materials i just suggested as far as i would like to go for the tools.

for those who want the explanation:
i had a suggestion but i wasnt sure where to put it. i am watching minetv ep3 and hearing the really cool changes coming in ee3 and the damage value philosopher stone (minium stone?) i got an idea. why not damage values for the base recipes of the dark matter and red matter tools, armor, and weapons. just give them a nice damage value and the 'op-ness' of them will be significantly reduced. if you want to keep the permanent tools like we have now you could have them craftable but much more expensive. my idea is that you could have the crafting table involve mulltiple of the damage value tools and perhaps a few repair talismans. my specific idea is a talisman surrounded by 8 of the tool would make an infinite tool. ie: dm pick costs 2 diamonds and 3 dark matter and has a damage value of 750, 8 damage value dm pick and 1 talisman of repair makes a inf dmpick. i go further with this in making the rm items more expensive by having them craftable with the inf dm items. same recipe and same idea as the dm damage and infinite items. you could keep the same abilities across each item as well (perhaps increase the damage by N for how many blocks are damaged ie: dm pick long mode 3x damage or dm hammer mega impact mode 9x damage). perhaps put a damage value on destruction catalyst and the other 2 like it. something like 8 or 16 uses to make it alot less attractive as an alternative to actual mining.
